job_creator.php

```diff
@@ -0,0 +1,239 @@
+<?php
+
+class JobCreator
+{
+    public function getInstallerVersion($githubRepository, $branch)
+    {
+        $repo = explode('/', $githubRepository)[1];
+        if (in_array($repo, NO_INSTALLER_REPOS)) {
+            return '';
+        }
+        $v = explode('.', $branch);
+        if (count($v) == 1) {
+            return '4.x-dev';
+        }
+        if (in_array($repo, LOCKSTEPED_REPOS)) {
+            return '4' . $v[1] . 'x-dev';
```

Comment:
```suggestion
            return $version[0] . $version[1] . 'x-dev';
```
They're lockstepped, so the major should always match as well. May as well make that change before we need it since it's an obvious one.

Reply:
Nope, because asset-admin and a bunch of others use `1`, while others such as framework and cms use `4`

Reply:
Ahh yup good point. I had a brain-dumb moment.